cfpb/cfgov-refresh
# HomePage model for new site root

## Additions

- Homepage Model

## Review

- @kurtw 
- @richaagarwal 

## Testing
- Create a new `HomePage` at root. This is page should only be createable at root
- Change `sites` root to new HomePage
![screen shot 2016-02-25 at 11 52 51 am](https://cloud.githubusercontent.com/assets/254877/13327066/594041b0-dbb6-11e5-990e-e2931909aebe.png)

- Publish page and view new root home page

## Checklist
